package.json

```diff
@@ -107,33 +107,33 @@
     "react-refresh": "^0.14.0",
     "react-test-renderer": "^18.2.0",
     "resolution-independence": "^1.0.0",
-    "resolve": "^1.22.1",
-    "sass-loader": "^13.2.0",
-    "semver": "^7.3.8",
-    "source-map-loader": "^3.0.1",
+    "resolve": "^1.22.2",
+    "sass-loader": "^13.2.2",
+    "semver": "^7.5.0",
+    "source-map-loader": "^4.0.1",
     "strip-ansi": "^6.0.1",
-    "style-loader": "^3.3.1",
+    "style-loader": "^3.3.2",
     "tar": "^6.1.13",
     "terser-webpack-plugin": "^5.3.7",
-    "validate-npm-package-name": "^3.0.0",
-    "webpack": "^5.76.0",
-    "webpack-dev-server": "^4.11.1"
+    "validate-npm-package-name": "^5.0.0",
+    "webpack": "^5.80.0",
```

Comment:
Please use the same version with dev-utils.

Reply:
OK